github-management/org-owners-guide.md

```diff
@@ -47,6 +47,21 @@ code in.
 It is easier to start new code in a Kubernetes organization than it is to
 transfer in existing code.
 
+## Licensing Guidance
+
+Section 11 of the [CNCF Charter] provides guidance on how to license code and
+documentation owned by the CNCF projects. However, it doesn't explicitly cover
+situations where a repo may have mixed content (such as repos that contain both
+code and documentation living alongside each other).
+
+For clarity, here is the guidance for Kubernetes GitHub organizations:
```

Comment:
```suggestion
For clarity, here is the guidance for Kubernetes GitHub repositories:
```

Reply:
The language here is consistent in tone with the rest of the document. The guidelines around licensing apply to the whole organization, and then the specific repo cases are outlined in the bullets below.

Reply:
sounds good. thanks @cblecker